.github/workflows/deploy.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install mdbook
+      run: |
+        mkdir mdbook
+        curl -Lf https://github.com/rust-lang/mdBook/releases/download/v0.4.8/mdbook-v0.4.8-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=./mdbook
```

Comment:
Hm, this will cause RFCs book to use 0.4.8 probably almost indefinitely this way. Maybe good, but definitely a different from the previous setup.

Reply:
I personally prefer to lock versions so that it doesn't break or otherwise have silent issues in the future.  I do not know of a way to get notifications for errors pushing to master. And some issues with mdbook could just be rendering problems where a manual inspection is required. This is also substantially faster to run.

However, if infra would prefer to update on each push, I can easily add that.

Reply:
Mkdir mdbook Curl - Lf http://github.com/rust-lang/mdbook/releases/download/v0. 4.8/mdbook-v0. 4.8-x86_64-known-linux-gnu.tar.gz|tar -xz --directory =>/mdbook . Gethub/workflow/deploy. Yml